bgpd/bgp_nb_config.c

```diff
@@ -2129,12 +2237,38 @@ int bgp_neighbors_neighbor_destroy(struct nb_cb_destroy_args *args)
 int bgp_neighbors_neighbor_local_interface_modify(
 	struct nb_cb_modify_args *args)
 {
+	struct bgp *bgp;
+	const char *peer_str;
+	const char *intf_str;
+	struct peer *peer;
+
 	switch (args->event) {
 	case NB_EV_VALIDATE:
 	case NB_EV_PREPARE:
 	case NB_EV_ABORT:
+		return NB_OK;
 	case NB_EV_APPLY:
-		/* TODO: implement me. */
+		bgp = nb_running_get_entry(args->dnode, NULL, true);
+		peer_str =
+			yang_dnode_get_string(args->dnode, "../remote-address");
+		peer = bgp_neighbor_peer_lookup(bgp, peer_str, args->errmsg,
+						args->errmsg_len);
+		if (!peer) {
+			snprintf(args->errmsg, args->errmsg_len,
```

Comment:
Just curious ... Should we be using `%pI4` instead of sprintf here?

Reply:
Northbound return error message in `errmsg` string storage form. 
Another thing peer is passed as string form \ instead of converting to v4/v6 address form using as is.

Reply:
Northbound return error message in `errmsg` string storage form. 
Another thing peer is passed as string form \ instead of converting to v4/v6 address form using as is.